core/api/rpc/json.hpp

```diff
@@ -1158,7 +1161,9 @@ namespace fc::api {
       Set(j, "Amount", v.amount);
       Set(j, "MinSettleHeight", v.min_close_height);
       Set(j, "Merges", v.merges);
-      Set(j, "SignatureBytes", v.signature_bytes);
+      const auto sig{v.signature_bytes.map(
+          [](auto &bytes) { return Signature::fromBytes(bytes).value(); })};
```

Comment:
`Signature::fromBytes()` could return an error. Segfault in this case

Reply:
It can return `outcome::result` with error.
But `.value()` won't segfault, it will throw `system_error` exception (used here in json and in cbor to simplify code), which is can be `catch`

Reply:
Ok